### PR TITLE
feat: added the navigation component

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
this pull request contains the navigation component of the component library.
Screenshot (118)

### CODE REVIEW

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+.standard-header{
+    height: 5rem;
+    width: 90%;
+    background-color: var(--bg-color);
+    display: flex;
+    justify-content: center;
+    padding-left: 1rem;
+    border-radius: 5px;
```
Comment (aliamaanansari) [inline components/navigation.css:11]: please use rem unit instead of pixel

File: components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    color: white;
```
Comment (aliamaanansari) [inline components/navigation.css]: please use var() for colors, dont hardcode them

File: components/navigation.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.standard-header .box-pill .btn{
+    color: white;
```
Comment (aliamaanansari) [inline components/navigation.css]: same here

**Review by aliamaanansari — CHANGES_REQUESTED** [review]

File: components/navigation.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply components/navigation.css:11]: for very small units it is advisable to use pixels instead of rems.

File: components/navigation.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply components/navigation.css]: will make the necessary changes

Comment (vaibhav-mahalle) [reply components/navigation.css]: made the necessary changes in the following commit.

File: components/navigation.css (hunk 3 above)
Comment (vaibhav-mahalle) [reply components/navigation.css]: made the necessary changes in the following commit.

**Review by AtulHande03a — COMMENTED** [review]
looks good to merge!!

**Pull request merged**